x/auth/tx/builder.go

```diff
@@ -62,6 +63,10 @@ func newBuilder(cdc codec.Codec) *wrapper {
 	}
 }
 
+func (w *wrapper) Size() int {
+	return len(w.bodyBz) + len(w.authInfoBz)
```

Comment:
This seems incorrect, as it doesn't take into account the signatures length. The correct way is `len(proto.Marshal(TxRaw{...}))`

Reply:
I agree with that, we may need hashes too.  I am moving this detail to #13262